gapis/resolve/memory.go

```diff
@@ -227,20 +268,20 @@ func Memory(ctx context.Context, p *path.Memory, rc *path.ResolveConfig) (*servi
 	var reads, writes, observed memory.RangeList
 	typedRanges := []*service.TypedMemoryRange{}
 
-	s.Memory.SetOnCreate(func(id memory.PoolID, pool *memory.Pool) {
-		if id == memory.PoolID(p.Pool) {
-			pool.OnRead = func(rng memory.Range, root uint64, id uint64, apiId coreid.ID) {
+	s.Memory.SetOnCreate(func(poolID memory.PoolID, pool *memory.Pool) {
+		if poolID == memory.PoolID(p.Pool) {
+			pool.OnRead = func(rng memory.Range, root uint64, typeID uint64, apiId coreid.ID) {
```

Comment:
Glad to see you upgrading the descriptiveness of preexisting names here!

Reply:
🎉